modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/sys/view/SqlAbstractSystemView.java

```diff
@@ -131,4 +136,9 @@ protected static Column newColumn(String name, int type) {
 
         return sql.toString();
     }
+
+    /** {@inheritDoc} */
+    @Override public String getDropSQL() {
```

Comment:
I think we shouldn't reregister system view.
We should return empty data when applicable instead of dropping it.